shell.nix

```diff
@@ -0,0 +1,56 @@
+scope@{ pkgs ? import <nixpkgs> { } }:
```

Comment:
I'd prefer to omit this from this PR.

Reply:
whoops, didn't mean to commit the nix stuff, sorry!